system_strings/sysmes_text.en

```diff
@@ -6501,47 +6550,71 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
 }
 [sha:6afce19b0e7f236968be889a51b0cb4a12ad48e9]{
 -- ……一緒にはいられない。^彼女が吸血鬼だからじゃない。^今の自分には、大切な人がいる。
-……一緒にはいられない。^彼女が吸血鬼だからじゃない。^今の自分には、大切な人がいる。
+// Ref 37614.
+// Copied a bit extra for context.
+// @flowchart_descr
+We can't be together...
+ Not because she's a vampire, but because right now, I have someone I value more than my own life.
 }
 [sha:cb72b0928c8c0839db1d4ecbdd7839fc586f0479]{
 -- アルクェイドが倒れる。^地面が彼女の血で染まっていく。^駆け寄ろうとした、その時――
 Arcueid has fallen. Her blood^stains the ground. I'm about to^run over to her, when―――
 }
 [sha:be5d924a295e911b8d7926f662f0370f132f556d]{
 -- 暴走するアルクェイド。^二人は学校まで撤退し、^戦いに備える。
-暴走するアルクェイド。^二人は学校まで撤退し、^戦いに備える。
+// Second half the same as 38f8a4fe
+// @flowchart_descr
+Arcueid has started to run rampant.
+ Ciel and I retreat to the school, and ready ourselves for battle.
 }
 [sha:0f34a856d20efd289e534ffe2ef37ae5cb6c2bd6]{
 -- 死のない真祖を絶命させる。^それを成すためには、^先に殺すべき物がある……！
-I need to kill the deathless^True Ancestor. But in order to^do that, I must kill something^else first!
+// @flowchart_descr
+I need to kill the deathless True Ancestor.
+ But in order to do that, I must kill something else first!
 }
 [sha:446370fd9a9c3560c24307456d0d4dee392d3435]{
 -- 彼女にナイフを突き立てる。^その傷が自分だけの物であれ、^と願いながら――
-I pierce her with my knife. I^pray that those scars are mine^alone to bear―――
+// @flowchart_descr
+I pierce her with my knife.
+ I pray that those scars are mine alone to bear―――
 }
 [sha:b75f12c5fc4b10f3143bdf364f5969ae08628f5f]{
 -- この命はじきに停止する。^だが、ロアの自由にはさせない。^残された方法はただひとつ。
-My life will soon be at an end.^But I can't allow Roa to roam^free――there is only one thing^left to do.
+// @flowchart_descr
+My life will soon be at an end.
+ But I can't allow Roa to roam free――there is only one thing left to do.
```

Comment:
first line is a modification of 38257
no changes necessary here, but maybe "this body" to "my body" there